releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -6,6 +6,33 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
 
+## 1.2.0 - January 13, 2020
+
+* Simplified Feature Management config file format.
+
+```yaml
+feature-management:
+  Beta: true
+  DarkTheme:
```

Comment:
This is for Java developers. If it makes more sense for a Java developer to use `data-theme` instead of `DarkTheme`, we should do so in our examples.

Reply:
This part here is the name of the feature, so I imange it would be like this inside of App Configuration, so if they exported to a file this is what they would get. Both would work here, I just imange this being more likely.